policyengine_uk/tests/policy/baseline/gov/dfe/universal_childcare_entitlement/universal_childcare_entitlement_eligible.yaml

```
@@ -0,0 +1,64 @@
+- name: Child aged 3 in England is eligible
+  period: 2024
+  input:
+    people:
```

similarly simplify as
```suggestion
    age: 3
    country: ENGLAND
output:
    universal_childcare_entitlement_eligible: True
```